cmd/snap/cmd_snap_op.go

```diff
@@ -991,6 +992,8 @@ func init() {
 		waitDescs.also(map[string]string{
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"revision": i18n.G("Remove only the given revision"),
+			// TRANSLATORS: This should not start with a lowercase letter.
+			"purge": i18n.G("Remove the snap without creating snapshot data"),
```

Comment:
"Remove the snap without creating a snapshot of the data" ?  cc @chipaca   or s/of the/of its/

Reply:
:+1: to @pedronis' suggestion, `Remove the snap without creating a snapshot of its data` (although it's only a small thing).

Reply:
Agreed either of those reads better, although @degville would know more.

One thing though, would be nice if we used the `save` verb, as that's what we use for snapshots (ie something like `Remove the snap without first saving a snapshot of its data`)

Reply:
(but the negation there feels weird to me)

Reply:
How about "Remove the snap without saving a snapshot of its data"? (note no "first")

Reply:
> (but the negation there feels weird to me)

Yeah, it does a little. It's a little cleaner without *first*:
`Remove the snap without saving a snapshot of its data`